docs/design_proposal.md

```diff
@@ -59,8 +59,8 @@ In previous technical challenges we mentioned two things: Memory consumption and
 As a prototype, Python is implemented in [current Repo](https://github.com/open-telemetry/opentelemetry-lambda/tree/main/python).
 
 We don’t provide SDK public layers for every language. This is required in the following two cases:
-Languages with auto-instrumentation support, such as Python and Java
-Scripting languages such as Python, Node.js, and Ruby
+Languages with auto-instrumentation support, such as Python and Java, and 
+Scripting languages such as Python, Node.js, and Ruby.
```

Comment:
Maybe we could do this as a bulleted list?

```suggestion
- Languages with auto-instrumentation support, such as Python and Java
- Scripting languages such as Python, Node.js, and Ruby
```

Reply:
No objectioon to that.